ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactClippingViewGroup.java

```diff
@@ -7,13 +7,11 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  */
 
-package com.facebook.react.views.view;
+package com.facebook.react.uimanager;
```

Comment:
I moved it to avoid a circular reference of dependencies when building with buck.